lib/Sema/TypeCheckAttr.cpp

```diff
@@ -306,18 +306,6 @@ class AttributeChecker : public AttributeVisitor<AttributeChecker> {
         }
       }
 
-      // We need isolation check here because global actor isolation
-      // could be inferred.
-
-      auto isolation = getActorIsolation(F);
-      if (isolation.isGlobalActor()) {
-        diagnoseAndRemoveAttr(
-            attr,
-            diag::attr_execution_concurrent_incompatible_with_global_actor, F,
-            isolation.getGlobalActor());
-        return;
```

Comment:
If we are not using `attr_execution_concurrent_incompatible_with_global_actor`, let's remove it?